runtimes/server-interface/logging.ts

```diff
@@ -1,7 +1,13 @@
+import { LogLevel } from '../runtimes/util/loggingUtil'
+
 /**
  * The logging feature interface
  */
 export type Logging = {
-    // TODO: Implement log levels
+    level: LogLevel
```

Comment:
Why is it part of server interface? Is it expected to be used by servers?

Reply:
We define method per each log level below - it should be enough for the server developer to know that we support log levels.

Reply:
What you describe is a part of runtime (LoggingServer is inside of runtimes, it's not real server). Server interface that you are extending here exposes `level` to the servers/capabilities that are implemented by service teams. It gives them possibility to read and also modify the logging level, which does not make sense.